Dockerfile-dind-rootless.template

```diff
@@ -1,3 +1,16 @@
+FROM golang:1.14-alpine{{ .alpine }} AS builder
+
+# https://github.com/rootless-containers/rootlesskit/releases
+ENV ROOTLESSKIT_VERSION 0.11.0
+
+RUN set -eux; \
+	wget -O rootlesskit.tgz "https://github.com/rootless-containers/rootlesskit/archive/v${ROOTLESSKIT_VERSION}.tar.gz"; \
+	mkdir -p "$GOPATH/src/github.com/rootless-containers/rootlesskit"; \
+	tar --extract --file rootlesskit.tgz --directory "$GOPATH/src/github.com/rootless-containers/rootlesskit" --strip-components 1; \
+	go build -o /usr/local/bin/rootlesskit github.com/rootless-containers/rootlesskit/cmd/rootlesskit; \
+	go build -o /usr/local/bin/rootlesskit-docker-proxy github.com/rootless-containers/rootlesskit/cmd/rootlesskit-docker-proxy; \
+	rootlesskit --version
```

Comment:
Why don't we use the rootlesskit binary included in https://download.docker.com/linux/static/test/x86_64/docker-rootless-extras-20.10.0-beta1.tgz ? (I think this was intentional but I forgot the reason)

@tianon @yosifkit

Reply:
I think it was probably just because rootless-extras for the released versions wasn't new enough?  If both 19.03.x (and 20.10.x) are new enough, I agree we could/should just pitch this entirely instead. :+1:

Reply:
Both should be new enough